src/main/java/Duke.java

```diff
@@ -49,52 +49,118 @@ public static void main(String[] args) {
+                System.out.println("\toh shucks! Gargamel is here..we gotta hide");
                 System.out.println(SEPARATOR);
-            } else {
-                switch (command) {
-                case "bye":
-                    // Exit bot
-                    isDone = true;
+                break;
+            case "done":
+                try {
+                    // Find item number and update item status
+                    int itemNum = Integer.parseInt(userInput.replaceAll("[^0-9]", ""));
+                    taskList[itemNum - 1].setDone();
 
                     // Print output
                     System.out.println(SEPARATOR);
-                    System.out.println("\toh shucks! Gargamel is here..we gotta hide");
+                    System.out.println("\tBrainy Smurf: aah another thing done\n");
+                    System.out.printf("\t%d. [%s][%s] %s\n", itemNum, taskList[itemNum - 1].getTaskIcon(), taskList[itemNum - 1].getStatusIcon(),
+                            taskList[itemNum - 1].getDescription());
                     System.out.println(SEPARATOR);
-                    break;
-                case "list":
+                } catch (Exception exception) {
+                    System.out.println(SEPARATOR);
+                    System.out.println("\tOops I can't find a Smurf to help you with your task.");
+                    System.out.println(SEPARATOR);
+                }
+                break;
+            case "list":
+                // Print output
+                System.out.println(SEPARATOR);
+                System.out.println("\t\"Tracker Smurf!! I need you here!!\"\n");
+                for (int i = 0; i < Task.getNumOfTasks(); i++) {
+                    System.out.println("\t" + taskList[i]);
+                }
+                System.out.println(SEPARATOR);
+                break;
+            case "todo":
+                try {
+                    // Add task
+                    String todoDescription = userInput.replaceFirst(command, "").strip();
+                    taskList[Task.getNumOfTasks()] = new Todo(todoDescription);
+
+                    // Print output
+                    System.out.println(SEPARATOR);
+                    System.out.println("\tHandy Smurf is here to give you a hand!\n");
+                    System.out.println("\tI have added: ");
+                    System.out.println("\t" + taskList[Task.getNumOfTasks() - 1]);
+                    System.out.println(SEPARATOR);
+                } catch (Exception exception) {
+                    System.out.println(SEPARATOR);
+                    System.out.println("\tOops I can't find a Smurf to help you with your task.");
+                    System.out.println(SEPARATOR);
+                }
+                break;
+            case "deadline":
+                try {
+                    // Add task
+                    String deadlineDescription = userInputSubstrings[1]
```

Comment:
I like how u broke down the line to keep it under the character limit

Reply:
Appreciate that